test/nodejs/route.js

```diff
@@ -14,6 +16,30 @@ test('route: routes Berlin', function(assert) {
     });
 });
 
+test('route: routes Berlin on MLD', function(assert) {
+    assert.plan(5);
+    var osrm = new OSRM({path: berlin_mld_path, algorithm: 'MLD'});
+    osrm.route({coordinates: [[13.43864,52.51993],[13.415852,52.513191]]}, function(err, route) {
+        assert.ifError(err);
+        assert.ok(route.waypoints);
+        assert.ok(route.routes);
+        assert.ok(route.routes.length);
+        assert.ok(route.routes[0].geometry);
+    });
+});
+
+test('route: routes Berlin on CoreCH', function(assert) {
+    assert.plan(5);
+    var osrm = new OSRM({path: berlin_corech_path, algorithm: 'CoreCH'});
+    osrm.route({coordinates: [[13.43864,52.51993],[13.415852,52.513191]]}, function(err, route) {
+        assert.ifError(err);
+        assert.ok(route.waypoints);
+        assert.ok(route.routes);
+        assert.ok(route.routes.length);
+        assert.ok(route.routes[0].geometry);
+    });
+});
```

Comment:
Urgh please merge into master asap, I'm already working on changing the tests to monaco